lisa/tools/nfs_client.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.operating_system import SLES, Debian, Redhat
+from lisa.tools import Firewall, Mount
+from lisa.util import LisaException
+
+
+class NFSClient(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def can_install(self) -> bool:
+        return True
+
+    def setup(
+        self,
+        server_ip: str,
+        server_shared_dir: str,
+        mount_dir: str,
+        protocol: str = "tcp",
+    ) -> None:
+        # run installation
+        self._install()
+
+        # stop firewall
+        self.node.tools[Firewall].stop()
+
+        # mount server shared directory
+        self.node.tools[Mount].mount(
+            disk_name=f"{server_ip}:{server_shared_dir}",
+            point=mount_dir,
+            type="nfs",
+            options=f"proto={protocol},vers=3",
+        )
+
+    def _install(self) -> bool:
+        if isinstance(self.node.os, Redhat):
+            self.node.os.install_packages("nfs-utils")
+        elif isinstance(self.node.os, Debian):
+            self.node.os.install_packages("nfs-common")
+        elif isinstance(self.node.os, SLES):
+            pass
+        else:
+            raise LisaException(f"{self.node.os} not supported")
```

Comment:
The message shows only one line in message. So it needs to specify what's not supported. There is an UnsupportedException. It needs to use that one, so the test case level can convert it to SkippedException.